app/webpack/reactjs/components/open_studios/open_studios_info_form.tsx

```diff
@@ -0,0 +1,150 @@
+import { ErrorMessage, Field, Form, Formik, FormikProps } from "formik";
+import { MauButton } from "@reactjs/components/mau_button";
+import { MauCheckboxField } from "@reactjs/components/mau_checkbox_field";
+import { MauTextField } from "@reactjs/components/mau_text_field";
+import { FieldError } from "@reactjs/components/field_error";
+import { OpenStudiosRegistration } from "@reactjs/components/open_studios/open_studios_registration";
+import { api } from "@js/services/api";
+import * as types from "@reactjs/types";
+import Flash from "@js/app/jquery/flash";
+import { camelizeKeys } from "humps";
+import React, { FC, useState } from "react";
+
+interface OpenStudiosInfoFormProps {
+  location: string;
+  artistId: number;
+  participant: types.OpenStudiosParticipant;
+  openStudiosEvent: types.OpenStudiosEvent;
+  onUpdateParticipant: (participant: OpenStudiosParticipant) => void;
+}
+
+function denullify(participant) {
```

Comment:
Not sure if this is a shared thing.  Seemed useful here to suppress Formik warnings about undefined states in form initial values.